djangofromscratch/JWTLogin/admin.py

```diff
@@ -8,4 +8,4 @@
 User = get_user_model()
 
 
-admin.register(User)
+admin.site.register(User)
```

Comment:
is it possible to show the user details on the listing page with some additional filters like search with username, and email?